rudder-web/src/main/webapp/style/newTheme/newTheme.css

```diff
@@ -0,0 +1,590 @@
+/*
```

Comment:
Can you move that file into /style/rudder ? We want Rudder specific file into the same directory !